src/main/java/com/rybka/service/exchange/BankAggregatorService.java

```diff
@@ -0,0 +1,46 @@
+package com.rybka.service.exchange;
+
+import com.rybka.constant.BankAggregatorConstants;
+import com.rybka.model.BankData;
+import lombok.Data;
+import lombok.SneakyThrows;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.springframework.stereotype.Service;
+
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@Service
+@Data
+public class BankAggregatorService {
+
+    @SneakyThrows
+    public void loadExchangeRatesFor(String currency) {
+        var bankExchangePage = Jsoup.connect(String.format(BankAggregatorConstants.BANK_AGGREGATOR_URL, currency)).get();
+        var bankDataList = parseExchangeRatesPage(bankExchangePage);
+    }
+
+    @SneakyThrows
```

Comment:
remove @SneakyThrows and don't use it in the production code(only tests)

Reply:
fixed